G/gh_cli/build_tarballs.jl

```diff
@@ -3,28 +3,41 @@
 using BinaryBuilder, Pkg
 
 name = "gh_cli"
-version = v"1.0.0"
+version = v"2.2.0"
 
 # Collection of sources required to complete build
 sources = [
-    GitSource("https://github.com/cli/cli.git", "b2e36a0979a06b94bf364552a856c166cd415234"),
-    DirectorySource("./bundled"),
+    GitSource("https://github.com/cli/cli.git", "a843cbd72813025817a2293a09b31c4597a3f655"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd $WORKSPACE/srcdir/cli/
-# Use newer termenv to work around issue while building for FreeBSD:
-# https://github.com/muesli/termenv/issues/17
-atomic_patch -p1 ../patches/use-termenv-v0.7.2.patch
-make
+
+export GO_LDFLAGS="-s -w"
+export CGO_ENABLED=0
+
+if [[ "${target}" == *-apple-* ]]; then
+    export GOARCH=darwin
+fi
+
+if [[ "${target}" == aarch64* ]]; then
+    export GOARCH=arm
+fi
+
+if [[ "${target}" == x86_64-* ]]; then
+    export GOARCH=amd64
+fi
```

Comment:
This is useless since we already do this for all platforms: https://github.com/JuliaPackaging/BinaryBuilderBase.jl/blob/73da943d3c302b8165729d057b3e98e6429c9f80/src/Runner.jl#L505-L515

Reply:
Thanks